google-cloud-storage/clirr-ignored-differences.xml

```diff
@@ -21,4 +21,9 @@
         <method>com.google.cloud.storage.BucketInfo$Builder deleteLifecycleRules()</method>
         <differenceType>7013</differenceType>
     </difference>
+    <difference>
+        <className>com/google/cloud/storage/BlobInfo$Builder</className>
+        <method>com.google.cloud.storage.BlobInfo$Builder setCustomTime(java.lang.Long)</method>
+        <differenceType>7013</differenceType>
```

Comment:
Are you prepared to bump the major version? If not, please don't do this. This check is here for a reason. We should not simply ad an exclude every time it says something is wrong.